frontend/app/src/locales/en.json

```diff
@@ -3715,6 +3719,7 @@
       "gas_limit": "Gas Limit",
       "gas_price": "Gas Price",
       "gas_used": "Gas Used",
+      "gwei": "Gwei",
```

Comment:
Was there any reason Gwei was extracted? It is not really translatable.

Reply:
So i think I should make const in that component with that string. Previously we put "gwei" as asset in <amount-input />, and it become error because no identifier found.

Reply:
Yes const makes more sense since way will never be translated